python-package/setup.py

```diff
@@ -40,20 +42,31 @@ def copy_files_helper(folder_name):
     if not os.path.isfile('./_IS_SOURCE_PACKAGE.txt'):
         copy_files_helper('include')
         copy_files_helper('src')
+        copy_files_helper('windows')
         if use_gpu:
             copy_files_helper('compute')
         distutils.file_util.copy_file("../CMakeLists.txt", "./lightgbm/")
         distutils.file_util.copy_file("../LICENSE", "./")
 
 
 def clear_path(path):
-    contents = os.listdir(path)
-    for file in contents:
-        file_path = os.path.join(path, file)
-        if os.path.isfile(file_path):
-            os.remove(file_path)
-        else:
-            shutil.rmtree(file_path)
+    if os.path.isdir(path):
+        contents = os.listdir(path)
+        for file in contents:
```

Comment:
look like `file` is a built-in, can you help rename it?

Reply:
@wxchan Sure.